server/server.mk

```diff
@@ -25,7 +25,7 @@ LIBMLDB_SOURCES:= \
 	bound_queries.cc \
 	script_output.cc \
 	forwarded_dataset.cc \
-	serial_function.cc
+	serial_function.cc \
```

Comment:
It reduces noise on adding a new source file as it's a one line diff.  But no, not required.